coding-concepts.js

```diff
@@ -5,55 +5,91 @@
 // First, predict the outcome based on your understanding of the code.
 // Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
 
-// --------------------INSTRUCTOR EXAMPLE: What will this log?
-const colors = ["tangerine", "magenta", "lilac", "daffodil"]
+// --------------------INSTRUCTOR EXAMPLE: What will this log?ss
+// const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // console.log(colors.push("indigo"))
 
-// a) Your answer:
+
+// a) Your answer: 
+
+// /5
+
 // b) Verify and explain:
 
+    // /I ran the code and it gave me back the valkue of 5 since we are pushing (adding) another 
+    // value is reading the entire value line and bringbacck a value versus if we asked for an index, 
+    // whcih we then would have gotten a 4 back.
+
+
 
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2022"
-// console.log(cohort.length)
+console.log(cohort.length)
+
+// a) Your answer: 
+
+// /I believe it will return 10
 
-// a) Your answer:
 // b) Verify and explain:
 
+// /the reutrn was 10 because the function read the length of our variable including the
+// space inside of the string "".
+
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+console.log(greeting[4])
+
+// a) Your answer: 
+
+    // It will return the "o" located in the string.
 
-// a) Your answer:
 // b) Verify and explain:
 
+    // When I ran in through the terminal, it did return the "o" as expected becuase I prompted the system to search for the fourth value placement and remember the placements are indexed beginning at 0. Information in Brackets means indexed readings.
+
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
-// console.log(languages[index])
+console.log(languages[index])
 
 // a) Your answer:
+
+        // / I believe it will return Ruby.
+
 // b) Verify and explain:
 
+        // When I ran the code it did return "Ruby" because it is reading the index of 1. Remember indexing starts reading values at 0 when in an array. In this case, Javascript is 0
+        // and Ruby is 1 indexed.
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
```

Comment:
The original console log would return an error since .toUpperCase() only works on strings and not on arrays.  You are 100% correct that you can run .toUpperCase() on the individual values inside the array either by bracket notation, or if you have a lot of values in the array you can use iteration such as a for loop.